src/test/java/io/tus/java/client/TestTusClient.java

```diff
@@ -210,6 +256,7 @@ public void testResumeOrCreateUpload() throws IOException, ProtocolException {
         assertEquals(uploader.getUploadURL(), new URL(mockServerURL + "/foo"));
     }
 
+
     @Test
     public void testResumeOrCreateUploadNotFound() throws IOException, ProtocolException {
```

Comment:
@Acconut can you tell me what this test is for ? I have an assumption, but I don't want to write here any wrong information

Reply:
This test ensures that when a 404 is returned for the HEAD request, the uploader does not error out directly but instead creates a new upload. Makes sense? :)

Reply:
Yes makes sense to me! Thank you :)